app/models/repository_rows_my_module.rb

```diff
@@ -0,0 +1,7 @@
+class RepositoryRowsMyModule < ActiveRecord::Base
```

Comment:
https://stackoverflow.com/questions/11590469/rails-naming-convention-for-join-table